js/index.js

```diff
@@ -43,9 +43,50 @@ logo.setAttribute('src', siteContent["nav"]["img-src"])
 
 const topNav = document.querySelectorAll('nav a');
 
-a[0].textContent = siteContent.nav.nav-item-1; 
-a[1].textContent = 
-a[2].textContent =
-a[3].textContent =
-a[4].textContent =
-a[5].textContent =
\ No newline at end of file
+topNav[0].textContent = siteContent['nav']['nav-item-1']; 
+topNav[1].textContent = siteContent['nav']['nav-item-2'];
+topNav[2].textContent = siteContent['nav']['nav-item-3'];
+topNav[3].textContent = siteContent['nav']['nav-item-4'];
+topNav[4].textContent = siteContent['nav']['nav-item-5'];
+topNav[5].textContent = siteContent['nav']['nav-item-6'];
+
```

Comment:
easier way to go through them all 

nav.forEach(e, i =>
e.textContent = siteContent['nav'][nav-item-${i}]
)

might be i + 1